eventsources/sources/github/start.go

```diff
@@ -253,6 +256,15 @@ func (router *Router) PostInactivate() error {
 
 	githubEventSource := router.githubEventSource
 
+	if githubEventSource.APIToken == nil {
```

Comment:
Adding `|| githubEventSource.Webhook.URL == ""` might be better.

Reply:
done